lib/std/os/linux.zig

```diff
@@ -1386,6 +1386,58 @@ pub fn madvise(address: [*]u8, len: usize, advice: u32) usize {
     return syscall3(.madvise, @ptrToInt(address), len, advice);
 }
 
+pub fn posix_fadvise(fd: fd_t, offset: off_t, len: off_t, advice: usize) usize {
```

Comment:
```suggestion
pub fn posix_fadvise(fd: fd_t, offset: u64, len: u64, advice: usize) usize {
```

The offset and length cannot be negative.